articles/comparing-graphical-user-interface-gui-and-command-line-interface-cli/index.md

```diff
@@ -0,0 +1,82 @@
+
+#### Ease of use
+
+GUI is easier to learn and use. This is because of its user-friendly interface. Users receive immediate visual feedback when dealing with GUI, while this is not obvious in CLI. It would help if you had a higher degree of memorization and familiarity to effectively navigate and operate devices powered by CLI. New users may have more difficulty operating a CLI than GUI.
+
+#### Multitasking
+
+CLI offers an environment for multitasking. But the ease and ability to perform multiple tasks at once on one screen is not as efficient as in a GUI environment.
+
+With GUI, users rely on windows to control, manipulate, view and toggle through multiple folders and programs with ease. You can, for example, perform multiple tasks using MS Word.
+
+#### Remote access
+
+A command-line interface allows users to access and manipulate files in another device or computer over a network. But this is not such a straightforward process; you must know the commands to do so, which might pose a challenge to new users.
+
+A graphical user interface also allows users to access another computer remotely. Navigating devices remotely in GUIs is more straightforward and requires little experience, unlike in CLIs. IT professionals use GUI to manage servers and access user computers remotely.
+
+#### Scripting
+
+Users of CLI need to master scripting syntax and commands. New users may find it difficult to create scripts.
+
+GUI provides the option to create scripts using [programming software](https://www.defit.org/programming-software/#). With programming software, one can write scripts or develop other software without necessarily knowing all the syntax and commands.
+
+Examples of programming software are compilers, interpreters, and debuggers. A combination of all these software is referred to as integrated development environments ([IDEs](https://www.redhat.com/en/topics/middleware/what-is-ide)).
+
+#### Speed
+
+CLI is more preferred by professionals looking for performance and speed. With GUI, one has to navigate through different icons. This makes GUI slow. But with CLI, one only needs to utilize the keyboard to navigate through the interface. This results in faster performance.
+
+Modern GUIs have improved their speed, but one still requires to use of the mouse and the keyboard to type. Considering that CLI requires only the keyboard, users find taking their hand off the keyboard to move the mouse pointer slower.
+
+#### Control
+
+GUI allows control over the operating system and files. But one still needs CLI to perform advanced tasks. This is because the command line interface allows all the control over the operating system and files. Furthermore, tasks become simple with CLI. One can create a script containing a few command lines and leave it to perform most of the tasks. Shortcuts in GUI do not fully support automation or scripting. This way, you find that a user has to manually repeat each action within the graphical user interface.
```

Comment:
Brilliant comparison between CLI and GUI. Consider adding a handful of advantages of CLI over GUI and vice versa to give the reader a summary of the same. Or drawbacks of each, since some of the advantages of each are covered in the comparison of the two.